packages/verification-core/tests/unit/CertifactionEthClient.test.js

```diff
@@ -1,163 +0,0 @@
-import axios from 'axios'
```

Comment:
What's the reason to remove that? Haven't executed it in a while, so no idea if it even works.

Reply:
Just because was all commented out. We could leave it but it's not run...

Reply:
Ok, probably makes sense to just remove it.